### PR TITLE
Expose WorkflowMapProvider for SDK [5.8]

### PR DESCRIPTION
**Pull request by kewalker**
**Note:** Creating as draft PR until 5.8.

Exposing WorkflowMapProvider from Mobile so we can cast to it and avoid the dynamic/GetType workarounds.

### CODE REVIEW

File: docs/workflow/tutorial-mobile-show-map-callout.mdx [hunk 1]
```diff
@@ -246,7 +248,7 @@ namespace App1.Workflow
             // ArcGIS runtime Map
             Map map = mapProvider.Map;
             // ArcGIS runtime MapView
-            MapView mapView = mapProvider.GetType().GetProperty("MapView").GetValue(mapProvider, null) as MapView;
+            MapView mapView = (mapProvider as WorkflowMapProvider).View as MapView;
```
Comment (jsiemensLatitudegeo) [inline docs/workflow/tutorial-mobile-show-map-callout.mdx:251]: You should always do a null check after doing a cast with the as operator. If you want to assert that the cast is expected to always be successful, you should use the traditional explicit cast expression (WorkflowMapProvider)mapProvider.

Comment (kewalker) [reply docs/workflow/tutorial-mobile-show-map-callout.mdx]: Opted for the explicit cast.

**Comment by kewalker** [comment]
@Noah-Silvera took a bit - had to update dependencies, etc. But all looks well:
image

**Comment by Noah-Silvera** [comment]
> @Noah-Silvera took a bit - had to update dependencies, etc. But all looks well:
> image

Awesome! Thanks for checking that!!

**Pull request merged**